tests/test_suite.py

```diff
@@ -38,12 +40,12 @@
                            required=True,
                            help=("Path to the Source DataStream on this "
                                  "machine which is going to be tested"))
-common_parser.add_argument("--benchmark-id",
-                           dest="benchmark_id",
-                           metavar="BENCHMARK",
+common_parser.add_argument("--ref-id",
```

Comment:
we use `xccdf-id` in oscap for this, it's not the best name but it would make sense to be consistent